templates/shared_services/nexus-cert/terraform/certificate.tf

```diff
@@ -0,0 +1,49 @@
+    password = "0000000000"
+  }
+
+  certificate_policy {
+    issuer_parameters {
+      name = "Self"
+    }
+    key_properties {
+      key_size   = 2048
+      exportable = true
+      key_type   = "RSA"
+      reuse_key  = false
+    }
+    secret_properties {
+      content_type = "application/x-pkcs12"
+    }
+  }
+
+  # The certificate will get replaced with a real one, so we don't want Terrafomr to try and revert it.
```

Comment:
nit: typo
```suggestion
  # The certificate will get replaced with a real one, so we don't want Terraform to try and revert it.
```

Reply:
Will correct but typo probably exists in terraform/core/appgateway also